Scroller/EditableWrapper.js

```diff
@@ -40,7 +40,10 @@ const EditableShape = PropTypes.shape({
 	onComplete: PropTypes.func.isRequired,
 	centered: PropTypes.bool,
 	css: PropTypes.object,
-	removeItemFuncRef: EnactPropTypes.ref
+	hideItemFuncRef: EnactPropTypes.ref,
+	longPressMode: PropTypes.bool,
```

Comment:
Let's name it `selectItemBy` in string type to make it clear.

Reply:
It's fixed.